### ISSUE TITLE
Use docker supplied tini

### ISSUE THREAD

**Issue by daniel-noland** [jasonwhite/rudolfs#51]
Noticed this while I was tidying up the code (I'm not trying to be nitpicky, just trying to get familiar with this code base because I plan to use it as a dependency for another project soon and I want to make sure I understand how it works).

https://github.com/jasonwhite/rudolfs/blob/9f2eaadc108b8d69185dc560c371a9d9f0115fc2/Dockerfile#L13-L19

I was going to take care of this if it is still desirable in your view @jasonwhite 

I have a PR ready but it is based on top of #46 so I'll just post the PR as draft till that is merged.  No need for more merge resolution noise :smile:

**Comment by jasonwhite** [comment]
Yes, this change would be very welcome! I expect most distros will have a relatively recent version of Docker with the `--init` option now.
